pipelinewise/cli/config.py

```diff
@@ -318,6 +319,13 @@ def save_tap_jsons(self, target, tap, extra_config_keys=None):
         # Generate tap inheritable_config dict
         tap_inheritable_config = utils.delete_empty_keys({
             'temp_dir': self.get_temp_dir(),
+            'tap_id': tap.get('id'),
+            'query_tag': json.dumps({
```

Comment:
How would ppw behave if the command `run_tap` is ran with config that doesn't have these new property?
From what I see, these changes are breaking and not backward compatible, unless all pipelines are stopped and their config are imported.

Reply:
It's still working with the old config without these parameters, but not tagging the queries.

fastsync:
* `tap_id` and `query_tags` are optional values. If not defined then the query_tag will be generated with None values. Test added by 

target-snowflake:
* `query_tag` is an optional parameter in target-snowflake. If not defined in the `config.json` then nothing will be tagged by target-snowflake. [This test](https://github.com/transferwise/pipelinewise-target-snowflake/blob/195e7d7d9cd67f635f3c5c355dd7d31fc92740df/tests/integration/test_target_snowflake.py#L1046) is targeting when `query_tag` is defined in `config.json`. All the other test cases in the file running without `query_tag`